src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -59,13 +59,14 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
     const pageCompiled = compilePage(page, contentData);
     const drupalPageDir = path.join('.', drupalUrl);
     const drupalFileName = path.join(drupalPageDir, 'index.html');
+    const layoutName = entityBundle || 'page';
```

Comment:
Is this an improvement or just for testing? If so, why?

Reply:
@jhonnyoddball we were getting build failures when the entityBundle was returning null, setting the default template to page fixed that